### PR TITLE
Outsider and Shadekin Changes

### PR DESCRIPTION
**Pull request by FluffMedic**
Oh boi, Jasper is making PRs with 'interesting' proposals.
Anyway, imma lay down or something.

## Changelog

:cl:
add: Added Anomaly role, aka new shadekin role
add: Added an outsider radio
qol: Shadekin and outsiders are now separate
/:cl:

### CODE REVIEW

**Comment by Kazzc** [comment]
Anomaly role~
I wonder we can make Riftwalkers as a future role.

**Comment by TheCaramelion** [comment]
> I wonder we can make Riftwalkers as a future role.
Oh I'll be keeping an eye on this...👁️

**Comment by FluffMedic** [comment]
I have been informed finer manipulation loss is horrid.
So need to double check that.

**Comment by tacoguy7765093** [comment]
Yeah getting rid of advanced tools is going to absolutely brick them, I could be wrong but a quick glance over was straight up disabled them from using even regular tools and even weapons which I'm not sure is a good idea.

Also is a shame that the 3 kin rule is going to be enforced, especially since they were very popular species and would have to play dibs, but that's a debate for another area

**Comment by FluffMedic** [comment]
> Anomaly role~ I wonder we can make Riftwalkers as a future role.

Screw it, weekend project time

**Comment by Kazzc** [comment]
Think we can increase the shadekin limit to 5?
At the time when I added a limit of 3 we were a smaller community, with how big the lobby can get these days 5 sounds more reasonable. 
I'll go ahead and update the shadekin guidelines to reflect that change.

**Comment by FluffMedic** [comment]
Alright cool
Will do that shortly, as well as some other modifications.

File: code/controllers/communications.dm [hunk 1]
```diff
@@ -194,7 +196,7 @@ var/list/ANTAG_FREQS = list(SYND_FREQ, RAID_FREQ)
 //Department channels, arranged lexically
 var/list/DEPT_FREQS = list(AI_FREQ, COMM_FREQ, ENG_FREQ, ENT_FREQ, MED_FREQ, SEC_FREQ, SCI_FREQ, SRV_FREQ, SUP_FREQ)
 
-var/list/OFFMAP_FREQS = list(TALON_FREQ, CSN_FREQ) //VOREStation Add
+var/list/OFFMAP_FREQS = list(TALON_FREQ, CSN_FREQ, OUT_FREQ) //VOREStation Add
```
Comment (Darlantanis) [inline code/controllers/communications.dm:199]: needs comment

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 2]
```diff
@@ -127,7 +127,7 @@
 	channels = list("Response Team" = 1, "Science" = 1, "Command" = 1, "Medical" = 1, "Engineering" = 1, "Security" = 1, "Supply" = 1, "Service" = 1)
 
 /obj/item/device/encryptionkey/omni		//Literally only for the admin intercoms
-	channels = list("Mercenary" = 1, "Raider" = 1, "Response Team" = 1, "Science" = 1, "Command" = 1, "Medical" = 1, "Engineering" = 1, "Security" = 1, "Supply" = 1, "Service" = 1)
+	channels = list("Mercenary" = 1, "Raider" = 1, "Response Team" = 1, "Science" = 1, "Command" = 1, "Medical" = 1, "Engineering" = 1, "Security" = 1, "Supply" = 1, "Service" = 1, "Outsider" = 1)
```
Comment (Darlantanis) [inline code/game/objects/items/devices/radio/encryptionkey.dm:130]: needs comment

File: code/game/jobs/jobs.dm [hunk 3]
```diff
@@ -55,6 +55,7 @@ var/const/ENTREPRENEUR		=(1<<15)
 
 var/const/OTHER				=(1<<10) //CHOMPStation Add
 var/const/NONCREW			=(1<<0) //CHOMPStation Add
+var/const/ANAMAOLY			=(1<<0) //CHOMPstation Add
```
Comment (Darlantanis) [inline code/game/jobs/jobs.dm:58]: ANOMALY

**Pull request merged**